README.yaml

```diff
@@ -0,0 +1,88 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-cloudfront-cdn
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-cloudfront-cdn
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-cloudfront-cdn.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-cloudfront-cdn"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-cloudfront-cdn.svg"
+    url: "https://github.com/cloudposse/terraform-aws-cloudfront-cdn/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
```

Comment:
Add https://github.com/cloudposse/terraform-aws-s3-website

Reply:
Add this: https://github.com/cloudposse/terraform-root-modules/tree/master/aws/docs

`description: "Reference implementation combining `terraform-aws-s3-website` with `terraform-aws-cdn`"